test/Test/Effect.hs

```diff
@@ -49,26 +35,30 @@ import Path.IO (createDirIfMissing, removeDirRecur)
 import ResultUtil (expectFailure)
 import System.Directory (getTemporaryDirectory)
 import System.Random (randomIO)
+import Test.Hspec (
+  Spec,
+  SpecWith,
+  expectationFailure,
+  fit,
+  it,
+  runIO,
+  shouldBe,
+  shouldContain,
+  shouldEndWith,
+  shouldMatchList,
+  shouldSatisfy,
+  shouldStartWith,
+  xit,
+ )
 import Test.MockApi (FossaApiClientMockC, MockApiC, runApiWithMock, runMockApi)
 import Text.Printf (printf)
+import Type.Operator (type ($))
 
-type EffectStack a =
-  FinallyC
-    ( ExecIOC
-        ( ReadFSIOC
-            -- The FossaApiClient must be outside of Diagnostics because it
-            -- depends on it.
-            ( FossaApiClientMockC
-                ( DiagnosticsC
-                    -- MockApiC must be inside Diagnostics so its state is not
-                    -- lost when diagnostics short-circuits on an error.
-                    ( MockApiC (IgnoreLoggerC (IgnoreStickyLoggerC (StackC IO)))
-                    )
-                )
-            )
-        )
-    )
-    a
+-- The FossaApiClient must be outside of Diagnostics because it
+-- depends on it.
+-- MockApiC must be inside Diagnostics so its state is not
+-- lost when diagnostics short-circuits on an error.
```

Comment:
I moved these comments out because the formatter was making this completely unreadable.